### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by wssn**

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.3s | 12.7s | ⚠️ +354ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.024 | 2.011 | -0.01 |
| / avg req/sec | 1235.06 | 1243.25 | ⚠️ +8.19 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.22 | 1.188 | -0.03 |
| /error-in-render avg req/sec | 2049.18 | 2105.23 | ⚠️ +56.05 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.1s | -295ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 74d36afe05c88587011b9dfc8de1e5edbb33b595

**Review by lfades — CHANGES_REQUESTED** [review]
Change the loading state to be at the moment `data` is needed, so let's instead do this:

```
{data ? (
  <div>Your favorite food is {data.food}.</div>
) : (
  <div>Loading...</div>
)}
```

File: examples/with-firebase-authentication/pages/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Index = () => {
     user ? ['/api/getFood', user.token] : null,
     fetcher
   )
-
+
```
Comment (lfades) [inline examples/with-firebase-authentication/pages/index.js:18]:
```suggestion

```

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 11.8s | 11.6s | -184ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.993 | 1.935 | -0.06 |
| / avg req/sec | 1254.24 | 1291.7 | ⚠️ +37.46 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.242 | 1.217 | -0.02 |
| /error-in-render avg req/sec | 2012.24 | 2054.66 | ⚠️ +42.42 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.5s | -193ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: f20a4725d99e74a974a7dcd47f5ab11db52ade5e

**Review by lfades — APPROVED** [review]
Thank you!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.3s | ⚠️ +63ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.055 | 2.031 | -0.02 |
| / avg req/sec | 1216.72 | 1230.82 | ⚠️ +14.1 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.303 | 1.261 | -0.04 |
| /error-in-render avg req/sec | 1919 | 1983.32 | ⚠️ +64.32 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.3s | 13.3s | -15ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..14a7.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.57 kB | 5.57 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.07 kB | 7.07 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | Wssn/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: e69c92263892a83ebce67fddf11fabca22d10b0f

**Pull request merged**